packages/storybook/stories/Setup.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+import indexText from '!!raw-loader!./snippets/ReactIndex.snippet.tsx';
+
+<Meta
+  id="setup-communication-react"
+  title="Setup"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Setting up your app to use Azure Communications UI Library
+
+Currently the Azure Communications UI library supports react versions `>= 16.8.0` or ` < 18`
+
+Using the UI library in react developers will need to downgrade the version of react that they are currently
+getting from `create-react-app`. This page talks about packages that need downgrading and files that should be edited to
+get you up and running.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Node.js](https://nodejs.org/) Active LTS and Maintenance LTS versions (Node 14.19.0 and above).
+- An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp).
+- An identity with both VoIP and Chat scopes. Generate an identity using the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/identity/quick-create-identity).
+
+## Setting Up
+
+UI Library requires a React environment to be setup. Next we will do that. If you already have a React App, you can skip this section.
+
+### Set Up React App
+
+We'll use the create-react-app template for this quickstart. For more information, see: [Get Started with React](https://reactjs.org/docs/create-a-new-react-app.html)
+
+```bash
+
+npx create-react-app your-ui-lib-application --template typescript
+
+cd your-ui-lib-application
+
+```
+
+### Editing your files
+
+Once the create script has finished you will need to edit your `index.tsx` file to a older version.
```

Comment:
The prose here is unclear.

    The latest `creat-react-app` template is incompatible with React 16/17 libraries.
    Replace the content of the generated `index.tsx` file with the following snippet
    so that it works with the target React versions:

    <Source code={indexText} />

    Note: Various tutorials in this documentation modify the contents of the generated `App.tsx` file.
    The `index.tsx` file should look identical for all the tutorials, as shown here.